R/LSBuffonsneedlecommon.R

```diff
@@ -0,0 +1,97 @@
+
+
+
+
+#input$obs: number of  observations
+#input$k: number of crosses
+#observations = 10
+#lengthRatio = 80
+simulate_throws <- function(observations, lengthRatio){
+  
+  d <- 5 # distance of two adjacent lines
+  l <- lengthRatio * d / 100 # length of needle
+  
+  #seed1 <- runif(1, 10, 100)
+  #seed <- input$goButton[[1]] + seed1
+  
+  #seed <- runif(1, 10, 100)
+  
+  # initialization
+  xe <- vector("numeric", observations)
```

Comment:
```suggestion
  xe <- numeric(observations)
```
same for the 3 other instances below.